dashboard/app/controllers/api/v1/sections_controller.rb

```diff
@@ -12,4 +12,50 @@ def index
   def show
     render json: @section.summarize
   end
+
+  # POST /api/v1/sections
+  # Create a new section
+  def create
+    # Validate login_type
+    # TODO: Push validation into model?
```

Comment:
I kept a lot of logic (validation, defaults) in the controller here to mimic the pegasus API and isolate changes from other consumers of the Section model, but long-term we probably want this logic pushed down into the model.  Please offer advice on doing that correctly!